public/pages/CreateMonitor/components/AssociateMonitors/AssociateMonitors.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Fragment, useState, useEffect, useCallback } from 'react';
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import MonitorsList from './components/MonitorsList';
+import { FormikCodeEditor } from '../../../../components/FormControls';
+import * as _ from 'lodash';
+import { isInvalid, hasError, validateExtractionQuery } from '../../../../utils/validate';
+
+const AssociateMonitors = ({
+  monitors,
+  options,
+  searchType = 'graph',
+  isDarkMode,
+  monitorValues,
+}) => {
+  const [graphUi, setGraphUi] = useState(searchType === 'graph');
+
+  const queryTemplate = {
+    sequence: {
+      delegates: [],
+    },
+  };
+
+  const delegatesToMonitors = (value) =>
+    value.sequence.delegates.map((monitor) => ({
+      label: '',
+      value: monitor.monitor_id,
+    }));
+
+  useEffect(() => {
+    if (monitors?.length) {
+      const value = { ...queryTemplate };
+      monitors.map((monitor, idx) => {
+        let delegate = {
+          order: idx + 1,
+          monitor_id: monitor.monitor_id,
+        };
+        value.sequence.delegates.push(delegate);
+        _.set(monitorValues, `associatedMonitor_${idx}`, {
+          label: monitor.monitor_name || '',
+          value: monitor.monitor_id,
+        });
+      });
+
+      _.set(monitorValues, 'associatedMonitorsEditor', JSON.stringify(value, null, 4));
```

Comment:
These strings are being reused, let's create enum or constants so that we don't cause regression due to typos later on

Reply:
Extracted.